app/graphql/resolvers/item2item_resolvers.py

```diff
@@ -0,0 +1,46 @@
+from typing import List, Optional
+
+import strawberry
+from strawberry.types import Info
+from strawberry import argument
+from strawberry.types import Info
+from typing_extensions import Annotated
+
+from app.data_providers.dispatch import Item2ItemDispatch
+from app.data_providers.item2item import Item2ItemRecommender
+from app.graphql.corpus_recommendation import CorpusRecommendation
+from app.graphql.util import get_field_argument
+from app.singletons import item2item_recommender
+
+DEFAULT_RECOMMENDATION_COUNT = 10
+
+
+async def resolve_syndicated(root, info: Info,
+        resolved_id: Annotated[strawberry.ID, argument(
+            description='Article ID')],
+        recommendation_count: Annotated[Optional[int], argument(
+            description='Maximum number of recommendations to return, defaults to 10')] = 10,
+) -> List[strawberry.ID]:
```

Comment:
I would return a `CorpusSlate` here:
```suggestion
) -> CorpusSlate:
```
This makes the return type more consistent across recommendation endpoints. The main difference is that this returns a list of `CorpusRecommendation`. Clients need to pass the `id` field of `CorpusRecommendation` in analytics events, and this object contains a `CorpusItem` object, which Apollo is able to decorate.

You could choose to return `List[CorpusRecommendation]`, but the Snowplow events that recommendation-api emits assume you either have a slate or a lineup of slates.

Reply:
I forgot to change this part, it was supposed to return `List[CorpusRecommendation]`. I looked at `CorpusSlate` and it is designed specifically for Home and doesn't fit Related recommendations. It's wrong and confusing to call any recommendations a slate, so let's keep the List[CorpusRecommendation] and communicate to whoever is responsible to change the analytics pipeline to process this.